src/components/Editable/useOnPaste.ts

```diff
@@ -29,6 +29,8 @@ const useOnPaste = ({
       // mobile Safari copies URLs as 'text/uri-list' when the share button is used
       const plainText = e.clipboardData.getData('text/plain') || e.clipboardData.getData('text/uri-list')
       const htmlText = e.clipboardData.getData('text/html')
+      // if the browser is controlled by automation, or the clipboard comes from em, set true.
```

Comment:
This deserves a bit more explanation. *Why* do we need to check for browser automation? **Because puppeteer does not allow setData with other MIME types**. That's the key piece of information.